python/ray/ml/examples/pytorch/torch_image_example.ipynb

```diff
@@ -0,0 +1,941 @@
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14522)\u001b[0m 2022-05-09 14:03:57,559\tINFO torch.py:98 -- Moving model to device: cpu\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14522)\u001b[0m 2022-05-09 14:03:57,559\tINFO torch.py:132 -- Wrapping provided model in DDP.\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14521)\u001b[0m [1,  2000] loss: 2.259\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14522)\u001b[0m [1,  2000] loss: 2.261\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14521)\u001b[0m [1,  4000] loss: 1.870\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14522)\u001b[0m [1,  4000] loss: 1.898\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14521)\u001b[0m [1,  6000] loss: 1.706\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14522)\u001b[0m [1,  6000] loss: 1.712\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14521)\u001b[0m [1,  8000] loss: 1.626\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14522)\u001b[0m [1,  8000] loss: 1.614\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14521)\u001b[0m [1, 10000] loss: 1.527\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14522)\u001b[0m [1, 10000] loss: 1.570\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14521)\u001b[0m [1, 12000] loss: 1.470\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14522)\u001b[0m [1, 12000] loss: 1.476\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14521)\u001b[0m [2,  2000] loss: 1.437\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14522)\u001b[0m [2,  2000] loss: 1.411\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14521)\u001b[0m [2,  4000] loss: 1.408\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14522)\u001b[0m [2,  4000] loss: 1.407\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14521)\u001b[0m [2,  6000] loss: 1.358\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14522)\u001b[0m [2,  6000] loss: 1.391\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14521)\u001b[0m [2,  8000] loss: 1.354\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14522)\u001b[0m [2,  8000] loss: 1.336\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14521)\u001b[0m [2, 10000] loss: 1.291\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14522)\u001b[0m [2, 10000] loss: 1.341\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14521)\u001b[0m [2, 12000] loss: 1.262\n",
+      "\u001b[2m\u001b[36m(BaseWorkerMixin pid=14522)\u001b[0m [2, 12000] loss: 1.265\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2022-05-09 14:06:04,518\tERROR checkpoint_manager.py:189 -- Result dict has no key: training_iteration. checkpoint_score_attr must be set to a key of the result dict. Valid keys are ['trial_id', 'experiment_id', 'date', 'timestamp', 'pid', 'hostname', 'node_ip', 'config', 'done']\n"
```

Comment:
cc @xwjiang2010 this is the same bug that Spotify ran into

Reply:
ah got it. let me take a look!

Reply:
fyi, https://github.com/ray-project/ray/issues/24593